docs/cn/advanced/Alluxio-Storage-Management.md

```diff
@@ -115,7 +115,7 @@ Alluxio支持自定义回收策略，已有的实现包括：
 
 ### 开启和配置分层存储
 
-在Alluxio中，使用[配置参数](Configuration-Settings.html)开启分层存储。使用如下配置参数可以指定Alluxio的额外存储层：
+在Alluxio中，使用[配置参数]({{ '/cn/operation/Configuration.html' | relativize_url }})开启分层存储。使用如下配置参数可以指定Alluxio的额外存储层：
```

Comment:
I don't think Configuration.html page has anything with Tiered Store. Maybe you should link to https://docs.alluxio.io/ee/user/stable/en/advanced/Alluxio-Storage-Management.html

Reply:
Or if this should be the list of config properties should it be https://docs.alluxio.io/ee/user/stable/en/reference/Properties-List.html?

Reply:
Looks at the original en docs, the url is correct, but his way of translation makes the url confusing